README.MD

```diff
@@ -25,7 +25,7 @@ The repository also includes:
 screenshot comes from)
 
 There are also a bunch of embryonic packages that are not worth mentioning yet
-but may be up to future development (Pyworld, blender integration, WorldGUI)
+, but may be with future development (Pyworld, blender integration, WorldGUI)
```

Comment:
What I want to say here is "There are some packages that I started to develop and I don't want to mention them in the readme, *I may continue to develop these packages in the future*"
And as I understand your correction it's more like "The package may be worth mentioning in the future" (but maybe it's okay to put it like that, I just want to check).

Reply:
Ah, I understand... You can "but might be developped further."